pulumi/Pulumi.production.yaml

```diff
@@ -15,6 +15,8 @@ config:
     runners:
       - facility: am6
         plan: c3.medium.x86
+      - facility: ams1
+        plan: c1.large.arm
```

Comment:
I didn't see any arm hosts available in am6, but there are some c1.large.arm's available in ams1.